src/App.js

```diff
@@ -1,13 +1,10 @@
-import React from 'react';
-import Calculator from './components/Calculator';
-import './components/Calculator.css';
+/* eslint-disable react/prefer-stateless-function, quotes */
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
+import React from "react";
+import Calculator from "./components/Calculator";
```

Comment:
- [ ] Converting the `App` component to a functional component would save you from using `eslint-disable` comment.

Reply:
Thank you for that information! 😄